TeachingRecordSystem/src/TeachingRecordSystem.Core/DataStore/Postgres/Models/Process.cs

```diff
@@ -0,0 +1,13 @@
+namespace TeachingRecordSystem.Core.DataStore.Postgres.Models;
+
+public class Process
+{
+    public static Guid DqtProcessUserId { get; } = Guid.Empty;
```

Comment:
What's this field for? Do we still need it?

Reply:
Yeh we need it - it's for processes that happened in DQT (so we can't attach a TRS user ID to them).